src/exchange/information/information.html

```diff
@@ -4,257 +4,205 @@
+                                        data-simplepopover-single="true">
+                                </button>
+                                <button class="oui-status" type="button"
+                                        data-translate="exchange_dashboard_diag_ptrv6_label"
+                                        data-ng-class="{'oui-status_success': ctrl.exchange.serverDiagnostic.isPtrV6Valid, 'oui-status_warning': !ctrl.exchange.serverDiagnostic.isPtrV6Valid}"
+                                        data-ng-if="ctrl.exchangeServiceInfrastructure.isDedicated() || ctrl.exchangeServiceInfrastructure.isDedicatedCluster()"
+                                        data-simplepopover="{{ctrl.exchange.serverDiagnostic.ptrv6Tooltip}}"
+                                        data-simplepopover-title="{{'exchange_dashboard_diag_ptrv6_tooltip_title' | translate}}"
+                                        data-simplepopover-placement="right"
+                                        data-simplepopover-single="true">
+                                </button>
+                            </oui-tile-description>
+                        </oui-tile-definition>
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_ssl_expiration' | translate}}"
+                                             data-ng-if="ctrl.exchange.sslExpirationDate">
+                            <oui-tile-description>
+                                <span data-ng-bind="ctrl.exchange.sslExpirationDate | date:'mediumDate'"></span>
+                                <oui-message data-type="info"
+                                             data-dismissable="false"
+                                             data-ng-if="ctrl.messageSSL"
+                                             class="oui-paragraph">
+                                    <span data-ng-bind="ctrl.messageSSL"></span>
+                                </oui-message>
+                            </oui-tile-description>
+                            <oui-action-menu data-compact
+                                             data-align="end"
+                                             data-disabled="ctrl.hasSSLTask"
+                                             data-ng-if="ctrl.shouldDisplaySSLRenewValue">
+                                <oui-action-menu-item data-on-click="ctrl.sslRenew()">
+                                    <span data-translate="exchange_action_renew_ssl"></span>
+                                </oui-action-menu-item>
+                            </oui-action-menu>
+                        </oui-tile-definition>
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_expiration' | translate}}"
+                                             data-ng-if="ctrl.displayRenewDate()">
+                            <oui-tile-description>
+                                <span data-ng-bind="ctrl.exchange.expiration | date:'mediumDate'"></span>
```

Comment:
Refers to comment : https://github.com/ovh-ux/ovh-module-exchange/pull/181/files#r229202742